src/sidebar/components/AnnotationView.js

```diff
@@ -56,7 +56,9 @@ function AnnotationView({ loadAnnotationsService, onLogin }) {
 
         // Make the full thread of annotations visible. By default replies are
         // not shown until the user expands the thread.
-        annots.forEach(annot => store.setExpanded(annot.id, true));
+        annots.forEach(annot =>
+          store.setExpanded(/** @type {string} */ (annot.id), true)
```

Comment:
This cast is asserting that the annotation does have an ID (ie. has been saved) in this context. The same applies for similar changes elsewhere.